api/api.go

```diff
@@ -47,13 +47,12 @@ func (c *Client) Fetch(args []string) ([]*Problem, error) {
 	case 0:
 		url = fmt.Sprintf("%s/v2/exercises?key=%s", c.XAPIHost, c.APIKey)
 	case 1:
-		language := args[0]
-		url = fmt.Sprintf("%s/v2/exercises/%s?key=%s", c.XAPIHost, language, c.APIKey)
+		trackID := args[0]
+		url = fmt.Sprintf("%s/v2/exercises/%s?key=%s", c.XAPIHost, trackID, c.APIKey)
 	case 2:
-		language := args[0]
+		trackID := args[0]
 		slug := args[1]
-		url = fmt.Sprintf("%s/v2/exercises/%s/%s", c.XAPIHost, language, slug)
-	default:
```

Comment:
Totally not - I think I must have made a mistake in resolving the merge conflicts I had. All is restored and up to date now, though!